src/ims/store/_db.py

```diff
@@ -581,6 +585,87 @@ def detachedReportEntries(txn: Transaction) -> Iterable[ReportEntry]:
     # Incidents
     ###
 
+    def _fetchIncidents(
+        self, eventID: str, txn: Transaction
+    ) -> Iterable[Incident]:
+        parameters: Parameters = dict(eventID=eventID)
+
+        report_entries = defaultdict[int, list[ReportEntry]](list)
+        txn.execute(self.query.incidents_reportEntries.text, parameters)
```

Comment:
Quote from the future (via GitHub Time Machine™):
> But @wsanchez, you are such a liar! Right here is `incidents_reportEntries`, which is your fault!

This is true. The `_` indicates a subcategory of some sort. In this case, it's `<tableName>_<queryName>`.

Reply:
Good to know!